.github/workflows/release.yaml

```diff
@@ -2,7 +2,9 @@ name: Release to PyPi
 
 on:
   release:
-    types: [created]
+    # `created` does not fire on draft releases. `published` fires
+    # when a draft release is published. having both reduces footguns.
+    types: [created, published]
```

Comment:
Could `created` AND `published` ever fire for a single release? Or when a release is edited?

Reply:
- good question; not by my reading of the admittedly skimpy docs, but if it is, I don't think it matters (the second one will fail, and we can go back and edit this -- ideally as we've discussed this is temporary anyway)
- `edited` is a separate event.